components/ide/jetbrains/backend-plugin/launch-dev-server.sh

```diff
@@ -6,11 +6,18 @@
 set -e
 set -o pipefail
 
+# Default Options
+DEBUG_PORT=0
```

Comment:
As there's already an argument allowing specifying the port, could we leave the debug port as 44444 by default, to match the Run Configuration file (`.run/backend-plugin.run.xml`)?

I guess during development we'll be usually debugging it, so it would save us from remembering to use the `-p 44444` argument.

```suggestion
DEBUG_PORT=44444
```